### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by abhishek17551**

### CODE REVIEW

File: pages/components/image-input/input.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+/* label input styles */
+
+.label-input, .radio-input, .checkbox-input{
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```
Comment (xii-vi) [inline pages/components/image-input/input.css:5]: Can we put two value at the same time?
Just asking.

File: pages/components/image-input/input.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        
+            <div class="page-wrapper">
+                <!-- Inputs -->
+                <h2>Inputs</h2>
+                <p class="mg-t-s">Inputs are necessary for user interaction with the website. Inputs are used to take information from the user, and then save the data to the server, accordingly.</p>
+                <!-- Label Inputs -->
+                <h4 class="mg-t-s">Label Inputs</h4>
+                <p class="mg-t-s"> Label Input has a label present at the top of the input field. They can be customized to show error error-message, and change color accordingly.</p>
+
+                <div class="label-input  flex-se-column padding-s mg-l-l mg-t-s">
```
Comment (xii-vi) [inline pages/components/image-input/input.html:52]: Flex-se-column??
Use more generic name

File: pages/components/image-input/responsive-images.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            <div class="page-wrapper">
+                <h1>Responsive Image</h1>
+                <p class="mg-t-s">Responsive Image Components are used to display Images on websites, which are screen responsive. 
+                    They expand upto the width of the container they are in.</p>
+                <!-- the vertical card -->
+                <div>
+                    <h3 class="mg-t-s">Responsive Card</h3>
+                    <div class="flex-center mg-t-s mg-m">
+                        <div class="image-container">
+                            <img src="../../../assets/res-img.jpg" alt="responsive-image" class="responsive-img">
```
Comment (xii-vi) [inline pages/components/image-input/responsive-images.html:54]: You have used responsive img class,but it is not available in the image.css

**Review by xii-vi — APPROVED** [review]
Left few Comments.
Resolve them and you can merge.

File: pages/components/image-input/input.html (hunk 2 above)
Comment (abhishek17551) [reply pages/components/image-input/input.html:52]: It means flex direction is column and justify content is space evenly.

File: pages/components/image-input/responsive-images.html (hunk 3 above)
Comment (abhishek17551) [reply pages/components/image-input/responsive-images.html:54]: responsive-img is part of card styles, hence not included here.

File: pages/components/image-input/input.html [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                    style="width: 100%; height: 428px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="line-divider"></div>
+
+                <!-- Radio  -->
+                <h4 class="mg-t-s">Radio</h4>
+                <p class="mg-t-s">Radio buttons allow the user to select one option from a set.</p>
```
Comment (sharmayatendra) [inline pages/components/image-input/input.html:78]: mg-t-s?

**Review by sharmayatendra — COMMENTED** [review]
looks good overall

File: pages/components/image-input/input.html (hunk 4 above)
Comment (abhishek17551) [reply pages/components/image-input/input.html:78]: short notation for margin-top-small

**Pull request merged**